polyfills/ResizeObserver/config.json

```
@@ -0,0 +1,24 @@
+{
+	"browsers": {
+		"chrome": "<64",
+		"edge": "*",
+		"edge_mob": "*",
+		"firefox": "*",
+		"ie": "*",
+		"opera": "*",
+		"safari": "*",
+		"android": "<67",
+		"firefox_mob": "*",
+		"ie_mob": "*",
+		"op_mob": "*",
+		"ios_saf": "*"
+	},
+	"repo": "https://github.com/que-etc/resize-observer-polyfill",
+	"docs": "https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver",
+	"spec": "https://drafts.csswg.org/resize-observer-1",
+	"license": "W3C",
```

@que-etc has the polyfill licensed as [MIT](https://github.com/que-etc/resize-observer-polyfill/blob/master/LICENSE)
```suggestion
	"license": "MIT",
```